docker/base-datadir/geth/chaindata/CURRENT

```diff
@@ -0,0 +1 @@
+MANIFEST-000003
```

Comment:
the `base-datadir` should show up in the `.gitignore` right?

Reply:
I don't think so? Why do you think this directory should be ignored? It's required to build `Dockerfile.dev-geth`.

Reply:
Oh shoot:
> The base datadir used to ensure that the genesis block is always the same.

Should have reviewed before mindless-ly posting.

Small suggestion - you should be able to put these files into a single `base-datadir.tar.xz` file [as Docker `ADD` will automatically extract tar archives](https://docs.docker.com/engine/reference/builder/#add).

> If `<src>` is a local tar archive in a recognized compression format (identity, gzip, bzip2 or xz) then it is unpacked as a directory.

This way, the "base-datadir" will show up as a single file in Git.

Reply:
Nice!

Reply:
https://github.com/cowprotocol/services/pull/1121/commits/6a5903b698e0c1cf6c90ceec2aa03a6e50394d54